static/src/users/User.vue

```diff
@@ -1,17 +1,61 @@
 <template>
-<div>
-  Demo User App {{ users }}
+<div class="card card-collapsed border-0 mb-0">
+  <div class="card-header border-0 pl-0" data-toggle="card-collapse">
+    <h5 class="card-title font-weight-bold"><a href="#"><i class="dropdown-icon fe fe-users"></i>Personnes ayant accès au contrôle</a></h5>
+  </div>
+  <div class="card-body">
+
+    <user-list :users=inspectorUsers() profile-type="inspector"></user-list>
+
+    <user-list :users=auditedUsers() profile-type="audited"></user-list>
+
+    <div data-toggle="card-collapse" class="text-center bg-blue cursor-pointer text-white" style="cursor: pointer;"><i class="fe fe-chevron-up"></i></div>
+  </div>
 </div>
 </template>
 
 <script lang="ts">
   import Vue from "vue";
+  import axios from "axios"
+  import VueAxios from "vue-axios"
+
+  import UserList from "./UserList.vue"
+
+  Vue.use(VueAxios, axios)
 
   export default Vue.extend({
+    props: {
+      controlId: Number
+    },
     data() {
       return {
-        users: [1, 2]
+        users: []
+      }
+    },
+    methods: {
+      getUsers() {
+        var urlParams = new URLSearchParams()
+        urlParams.set("controls", this.controlId)
+        Vue.axios.get('/api/user/?' + urlParams.toString()).then((response) => {
+          this.users = response.data
+        })
+      },
+      auditedUsers() {
+        return this.users.filter(item => {
+           return item.profile_type == 'audited'
```

Comment:
utilise `===` (sauf si tu veux vraiment faire une égalité indépendante des types, mais ca fait des surprises en JS)
https://stackoverflow.com/questions/3586775/what-is-the-correct-way-to-check-for-string-equality-in-javascript

Reply:
ok